src/wallet/wallet.cpp

```diff
@@ -2564,7 +2565,7 @@ CWallet::Balance CWallet::GetBalance(const int min_depth, const bool avoid_reuse
             }
             ret.m_mine_immature += pcoin->GetImmatureCredit();
             ret.m_watchonly_immature += pcoin->GetImmatureWatchOnlyCredit();
-            if (CCoinJoinClientOptions::IsEnabled()) {
+            if (cj_enable) {
```

Comment:
312319f4d3de1ce3d71cb90d8d213119e1cdbb79:
```suggestion
            if (cj_enabled) {
```